auth-api/src/auth_api/models/org.py

```diff
@@ -48,6 +48,7 @@ class Org(BaseModel):  # pylint: disable=too-few-public-methods
     invitations = relationship('InvitationMembership', cascade='all,delete,delete-orphan', lazy='select')
     products = relationship('ProductSubscription', cascade='all,delete,delete-orphan', lazy='select')
     access_type = Column(String(250), index=True, nullable=True)  # for ANONYMOUS ACCESS
+    billable = Column('billable', Boolean(), default=False)
```

Comment:
Billable default should be True (all the accounts are billable except government accounts I guess). For dir search account it will be no-billable (billable=False). Also include a migration to add the value True for all existing accounts.

Reply:
sure..will do